### PR TITLE
Optimization: Categorize cmd modules by type

### PR DESCRIPTION
**Pull request by Stevent-fei**
### Describe what this PR does / why we need it
1. Categorize cmd modules by type
2. Divided into cluster module and image module by type

### CODE REVIEW

File: cmd/sealer/cmd/cluster/cluster.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cluster
+
+import "github.com/spf13/cobra"
+
+func NewCmdCluster() *cobra.Command {
+	cmd := &cobra.Command{
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/cluster/cluster.go:20]: no need to add `cluster` cmd.  just add those sub cmd to root cmd.

File: cmd/sealer/cmd/image/image.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+import "github.com/spf13/cobra"
+
+func NewCmdImage() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "image",
+		Short: "image sub-commands",
+	}
+	cmd.AddCommand(NewBuildCmd())
+	cmd.AddCommand(NewCompletionCmd())
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/image.go:25]: NewCompletionCmd, NewCompletionCmd is not belong to image

File: cmd/sealer/cmd/image/image.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package image
+
+import "github.com/spf13/cobra"
+
+func NewCmdImage() *cobra.Command {
+	cmd := &cobra.Command{
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/image.go:20]: just add those cmd to root cmd.

File: cmd/sealer/cmd/image/image.go [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+	cmd.AddCommand(NewInspectCmd())
+	cmd.AddCommand(NewLoadCmd())
+	cmd.AddCommand(NewLoginCmd())
+	cmd.AddCommand(NewPullCmd())
+	cmd.AddCommand(NewPushCmd())
+	cmd.AddCommand(NewRmiCmd())
+	cmd.AddCommand(NewSaveCmd())
+	cmd.AddCommand(NewSearchCmd())
+	cmd.AddCommand(NewTagCmd())
+	cmd.AddCommand(NewVersionCmd())
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/image.go:37]: same issue: versionCMD is not belong to image

File: cmd/sealer/cmd/cluster/apply.go [hunk 5]
```diff
@@ -23,26 +23,27 @@ import (
 
 var clusterFile string
 
-// applyCmd represents the apply command
-var applyCmd = &cobra.Command{
-	Use:   "apply",
-	Short: "apply a Kubernetes cluster via specified Clusterfile",
-	Long: `apply command is used to apply a Kubernetes cluster via specified Clusterfile.
+var longNewApplyCmdDescription = `apply command is used to apply a Kubernetes cluster via specified Clusterfile.
```
Comment (bxy4543) [inline cmd/sealer/cmd/cluster/apply.go:26]: const

File: cmd/sealer/cmd/image/image.go [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package image
+
+import "github.com/spf13/cobra"
+
+func NewCmdImage() *cobra.Command {
```
Comment (kakaZhou719) [inline cmd/sealer/cmd/image/image.go:19]: NewImageCommands()[]*cobra.Command{}

File: cmd/sealer/cmd/cluster/check.go [hunk 7]
```diff
@@ -29,29 +29,30 @@ type CheckArgs struct {
...
+var longNewCheckCmdDescription = `check command is used to check status of the cluster, including node status
+, service status and pod status.`
+
+// NewCheckCmd pushCmd represents the push command
+func NewCheckCmd() *cobra.Command {
+	checkCmd := &cobra.Command{
+		Use:     "check",
+		Short:   "check the state of cluster",
+		Long:    longNewCheckCmdDescription,
+		Example: `sealer check --pre or sealer check --post`,
```
Comment (justadogistaken) [inline cmd/sealer/cmd/cluster/check.go:41]: `
sealer check --pre \n
sealer check --post
`

File: cmd/sealer/cmd/cluster/check.go [hunk 8]
```diff
@@ -38,7 +38,7 @@ func NewCheckCmd() *cobra.Command {
 		Use:     "check",
 		Short:   "check the state of cluster",
 		Long:    longNewCheckCmdDescription,
-		Example: `sealer check --pre or sealer check --post`,
+		Example: `sealer check --pre \n sealer check --post`,
```
Comment (justadogistaken) [inline cmd/sealer/cmd/cluster/check.go:41]: I mean change line at "\n", not change or to "\n"

Comment (Stevent-fei) [reply cmd/sealer/cmd/cluster/check.go]: sorry, didn't look carefully😢

**Review by justadogistaken — APPROVED** [review]
LGTM.
@kakaZhou719 PTAL

**Pull request merged**